modules/imgcodecs/perf/perf_png.cpp

```diff
@@ -58,6 +61,30 @@ PERF_TEST(PNG, encode)
     SANITY_CHECK_NOTHING();
 }
 
+PERF_TEST_P(PNG, params,
+    testing::Combine(
+        testing::Values(IMWRITE_PNG_STRATEGY_DEFAULT, IMWRITE_PNG_STRATEGY_FILTERED, IMWRITE_PNG_STRATEGY_HUFFMAN_ONLY, IMWRITE_PNG_STRATEGY_RLE, IMWRITE_PNG_STRATEGY_FIXED),
+        testing::Values(IMWRITE_PNG_FILTER_NONE, IMWRITE_PNG_FILTER_SUB, IMWRITE_PNG_FILTER_UP, IMWRITE_PNG_FILTER_AVG, IMWRITE_PNG_FILTER_PAETH, IMWRITE_PNG_FAST_FILTERS, IMWRITE_PNG_ALL_FILTERS),
+        testing::Values(1, 6, 9)))
```

Comment:
@asmorkalov to reduce execution time i think `testing::Values(1, 6)` will give enough information.
9 is about 5-6 times slower than 6 and gives only about %1 lower file size.